GSServer.py

```diff
@@ -127,11 +120,10 @@ def verify_map_file(map_file, lanelet_map:LaneletMap):
     parser.add_argument("-s", "--scenario", dest="gsfiles", nargs='*', metavar="FILE", default="", help="GeoScenario file. If no file is provided, the GSServer will load a scenario from code")
     parser.add_argument("--verify_map", dest="verify_map", metavar="FILE", default="", help="Lanelet map file")
     parser.add_argument("-q", "--quiet", dest="verbose", default=True, help="don't print messages to stdout")
-    parser.add_argument("-n", "--no-dash", dest="no_dash", action="store_true", help="run without the dashboard")
+    parser.add_argument("-n", "--no_dash", dest="no_dash", action="store_true", help="run without the dashboard")
     parser.add_argument("-m", "--map-path", dest="map_path", default="", help="Set the prefix to append to the value of the attribute `globalconfig->lanelet`")
     parser.add_argument("-b", "--btree-locations", dest="btree_locations", default="", help="Add higher priority locations to search for btrees by agent btypes")
-    parser.add_argument("-wi", "--wait-for-input", dest="wait_for_input", action="store_true", help="Wait for the user to press [ENTER] to start the simulation")
-    parser.add_argument("-wc", "--wait-for-client", dest="wait_for_client", action="store_true", help="Wait for a valid client state to start the simulation")
+    parser.add_argument("--dash-pos", default=[0,0,0,0], dest="dash_pos", type=float, nargs=4, help="Set the position of the dashboard window (x y width height)")
```

Comment:
I would use empty list `[]` as the default. That clearly means that no values were provided. Otherwise, the user could have specified 0, 0, 0, 0, which is stupid but possible.